actions/methods.go

```diff
@@ -46,6 +47,7 @@ func MethodNotAllowed(c *gin.Context) {
 func GetSearchQueryParameters(c *gin.Context) (*datastore.QueryParams, *engine.Metadata, *map[string]interface{}, error) {
 	var requestParameters SearchRequestParameters
 	if err := c.Bind(&requestParameters); err != nil {
+		err = errors.New("error occurred while binding request parameters" + err.Error())
```

Comment:
use `fmt.Errof("somethingsomething %w", err)`

Reply:
switched to fmt.Errorf() wrapping